### PR TITLE
[#31][UI] As a user, I can response to dropdown survey type

### LINKED ISSUE #31: [UI] As a user, I can response to dropdown survey type
## Why
The application supports various types of survey items. After the user fills in all required survey items, the survey can be submitted and mark the survey as completed.

## Acceptance Criteria
- Display answer section for `dropdown` survey type
image
- Store the user's response for survey submission

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #31 

## What happened 👀

- Add select survey type.
- Update redirect path when leaving survey.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/14077479/180698339-ee8f4c2b-8a2d-4c69-8880-e8e940db0c77.mov

### CODE REVIEW

**Comment by malparty** [comment]
I think the label is wrong 😇 
image

**Review by malparty — APPROVED** [review]
Looks good to me! 
At first I was surprise to see `scss` in a project that used only css before, but in the end... why not! 😇

**Comment by hanam1ni** [comment]
> Looks good to me! At first I was surprise to see `scss` in a project that used only css before, but in the end... why not! 😇

@malparty To give you more context. At first, I think `CSS` is enough because this project heavily uses Tailwindcss, and if we need any `scss` feature, we can use CSS with PostCSS instead. 
But for CSS modules, it can be a lot more complex to hook PostCSS to the CSS module, and with Next.js built-in SCSS support, I decided to use SCSS 😃

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "jest": "27.4.7",
     "jest-canvas-mock": "2.4.0",
     "postcss": "8.4.5",
+    "sass": "1.54.0",
```
Comment (hoangmirs) [inline package.json:67]: Do you need to use `sass`? Could we just use only postCSS?
If you just need to use nested, you can try [Tailwind Nesting](https://tailwindcss.com/docs/using-with-preprocessors#nesting) plugin

**Review by hoangmirs — APPROVED** [review]
LGTM, just a small comment/suggestion

File: package.json (hunk 1 above)
Comment (hanam1ni) [reply package.json:67]: I use `sass` with the CSS module, and it's simpler compared to setup PostCSS to work with the CSS module.

**Pull request merged**